srcs/class/response.cpp

```diff
@@ -6,7 +6,7 @@
 #include <sys/wait.h>
 #include "unistd.h"
 
-Response::Response( int client_socket, webserv_conf &conf, Request const &req ) : conf(conf), req( req ) {
+Response::Response( int client_socket, Webserv_conf &conf, Request const &req ) : conf(conf), req( req ) {
```

Comment:
Du linting au passage!
```suggestion
Response::Response( int client_socket, Webserv_conf &conf, Request const &req ) : conf(conf), req( req )
{
```